src/app/base.ts

```diff
@@ -138,62 +140,74 @@ export const CommonHeaders = [
     "Upgrade",
     "Via",
     "Warning"
-]
-
-let today = new Date();
-let nextWeek = new Date();
-nextWeek.setDate(today.getDate()+7);
-
-const Tokens = {
-    "{user-mail}": "annew@CIE493742.onmicrosoft.com",
-    "{group-id}": "8f4e3cfd-432d-4b9a-b801-f424aaf08ca1",
-    "{drive-item-id}": "01ZDJCYOZPW7IKQNDL3NHZVRODY2GC2YKW",
-    "{section-id}": "1-fb22b2f1-379f-4da4-bf7b-be5dcca7b99a",
-    "{notebook-id}": "1-fb22b2f1-379f-4da4-bf7b-be5dcca7b99a",
-    "{group-id-with-plan}": "d2b6c7fe-f440-446b-99d8-9ac12e036bf0",
-    "{plan-id}": "RVBp6oJJt0K5f6Lq42zBK2UAA-Rs",
-    "{task-id}": "9jBI6WHDwk60lEWpL4TQ92UAGLhh",
-    "{extension-id}": "com.contoso.roamingSettings",
-    "{host-name}": "cie493742.sharepoint.com",
-    "{server-relative-path}": "sites/Contoso/Operations/Manufacturing",
-    "{group-id-for-teams}": "d2b6c7fe-f440-446b-99d8-9ac12e036bf0",
-    "{channel-id}": "ee3f919c-28e1-4659-86d2-8e37e581335c",
-    "{today}": today.toISOString(),
-    "{next-week}": nextWeek.toISOString(),
-    "AUTHENTICATED_DOMAIN": () => {
-        try {
-            return AppComponent.explorerValues.authentication.user.emailAddress.split("@")[1];
-        } catch(e) {
-            return "example@contoso.com"
-        }
-    },
-    "FULL_USER_EMAIL": () => {
-        try {
-            return AppComponent.explorerValues.authentication.user.emailAddress;
-        } catch(e) {
-            return "example@contoso.com"
-        }
-    }
+];
+
```

Comment:
Good points, added comments. We'll also update the README for 3rd and 1st party devs who want to add new queries with tokens